storagenode/nodenotifications/notifications.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package notifications
+package nodenotifications
```

Comment:
Why dont we use the same package as in the other PR?
I feel like this needs to be part of the notification package.

Reply:
He thought that you comment was about not having two packages with same name, honestly me as well

quote:
> Lets avoid a second package name with similar name in regards to PR #3659

Could you please explain what is your concern?